packages/portal-proto/src/features/facets/utils.unit.test.ts

```diff
@@ -501,3 +502,17 @@ describe("test years to days conversion", () => {
     expect(adjustYearsToDays(90, "years")).toEqual(32873);
   });
 });
+
+describe("test days to years conversion", () => {
+  test("days to years", () => {
```

Comment:
Can you add more test cases to cover:
range, age, year, percent?
Just to complete the coverage of the possible values of units. One each would be fine. If `adjustDaysToYears` adds  a case for unit=="percent" the test pick it up.

Reply:
sounds good. added a few for each of those units